pkg/volume/iscsi/iscsi.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2015 The Kubernetes Authors All rights reserved.
+Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
Sorry, I meant please change the copyright year from 2015 in the new files to 2016. Leave the existing files as is.